src/management/metadata.ts

```diff
@@ -0,0 +1,46 @@
+export interface FirebaseProjectMetadata {
```

Comment:
I'm curious of your thoughts since you're doing a lot of typing work here...

This file is all types. Generally, in TS, these probably should end up in a `.d.ts` file, right? But, we've avoided that by putting these types with the files that should "own" the types. I see these are used only in a couple places and I definitely get the value of having a separate file with them (but this seems like it's wrong and not either of those directions).

Does this make sense to make this file a `.d.ts` to specifically make this a "types" file (and set the president for those files)? Or do you think these types can be brought into other files? Is there an advantage you could see either way?

Reply:
I agree that I should rename it to a `.d.ts` file. Anyways this PR is meant to be merged after https://github.com/firebase/firebase-tools/pull/1470. I think I answer your question there.

Reply:
Resolved with the introduction of `projects.ts`